src/pages/rekey-reason/rekey-reason.scss

```diff
@@ -1,4 +1,67 @@
 page-rekey-reason {
+
+  .transferInfo {
+    color: map-get($colors-gds, gds-grey-1);
+    font-weight: initial;
+    letter-spacing: initial;
+  }
+
+  .staffNumber {
+    max-width: 40%;
+    width: 40%;
+    min-width: 40%;
+  }
+
+  .staffNumberRow {
+    height: 70px;
+    max-height: 70px;
+  }
+
+  ion-spinner {
+    margin-left: 6px;
+    * {
+      stroke: black;
+    }
+
+    &.spinner-circles circle,
+    &.spinner-bubbles circle,
+    &.spinner-dots circle {
+      fill: black;
+    }
+  }
+
+  tick-indicator {
+    margin-top: 12px;
+    margin-left: 5px;
+    div {
+      width: 40px;
+      height: 40px;
+      min-width: 22px;
+      .tick-background {
+        background-color: map-get($colors-gds, 'gds-grey-3');
+        color: map-get($colors, mes-white);
+        border-radius: 30px;
+        &.ticked{
+          background-color: map-get($colors-gds, 'gds-green');
+        }
+        span {
+          display: flex;
+          justify-content: center;
+          height: 100%;
+          .tick-icon {
+            &.icon-ios {
+              margin: 2px 0 0 2px;
+            }
+            font-size: 40px;
+            font-weight: 700;
```

Comment:
Font styling should not be specified here. Sizes and styles are inherited from sass typography partial and change with iPad accessibility zoom setting.